### DISCUSSION TITLE
Bug: Org website replacing current Discussions tab instead of opening in a new tab

### DISCUSSION THREAD

**Discussion by jfoliveira** [community/community#81495] [Discussions]
### Select Topic Area

Bug

### Body

Hi,

We're setting up discussions for our organization at https://github.com/orgs/devopness/discussions
The organization website is displayed correctly with an icon indicating that the website will be open in a separate tab, as expected.

However, when clicking the organization website it is being opened in the current tab, taking user out of Discussions.

I believe this is a bug on GitHub Discussions.
If that's by design/not a bug, could you please let us know how to modify that behaviour so our users do not lose focus of current `Discussions` tab when clicking to visit our website?

See images below for more context.

<img width="270" alt="image" src="https://github.com/community/community/assets/1943318/c7cb1c6d-f155-4dc1-a5ed-e471fb8de3ce">

![image](https://github.com/community/community/assets/1943318/8db908f5-aecb-4fef-9ea1-646dad6713c0)

**Comment by BilalBoudjema** [comment]
# Bug Report: GitHub Discussions Link Behavior

## Description
Encountering an issue in GitHub Discussions where the organization website link opens in the same tab instead of a new one, despite an icon indicating it should open in a separate tab.

## Steps to Address the Issue

### 1. Report the Bug
- Create an issue in the [GitHub Feedback discussions](https://github.com/github/feedback/discussions/categories/discussions).
- Provide a clear outline of the issue, steps to reproduce, and relevant screenshots.

### 2. Check for Updates
- Ensure your browser and any GitHub-related extensions are up-to-date.

### 3. Alternative Workaround
- Consider advising users to right-click and open the link in a new tab as a temporary solution.

### 4. Consult GitHub Documentation
- Review GitHub's Discussions documentation for any relevant settings or configurations.

### 5. Seek Community Input
- Post the issue in GitHub community forums for insights or solutions from other users.

### 6. Monitor GitHub Updates
- Stay updated with GitHub’s releases for any fixes or updates regarding this issue.

### 7. Contact GitHub Support
- If the issue persists, contact GitHub Support directly for assistance.

## Additional Notes

- Provide detailed information when reporting to help GitHub address the problem efficiently.

**Comment by jfoliveira** [comment]
Hi @BilalBoudjema 

> Create an issue in the [GitHub Feedback discussions](https://github.com/github/feedback/discussions/categories/discussions).

I don't understand. The link you posted to report an issue points to this category, so creating an issue here is exactly what I've done.

Are you a member of `GitHub Discussions` team?
Could you please be clear on what else you're expecting from my side to get a fix for this bug?
